README.rst

```diff
@@ -19,7 +24,81 @@ Usage
 ::
 
     tutor plugins enable mfe
+    tutor local quickstart
+
+Account
+~~~~~~~
+
+.. image:: https://raw.githubusercontent.com/overhangio/tutor-mfe/master/screenshots/account.png
+    :alt: Account MFE screenshot
+
+An MFE to manage account-specific information for every LMS user. Each user's account page is available at ``http(s)://{{ MFE_HOST }}/account``. For instance, when running locally: https://apps.local.overhang.io/account.
+
+Gradebook
+~~~~~~~~~
+
+.. image:: https://raw.githubusercontent.com/overhangio/tutor-mfe/master/screenshots/gradebook.png
+    :alt: Gradebook MFE screenshot
+
+This instructor-only MFE is for viewing individual and aggregated grade results for a course. To access this MFE, go to a course 🡒 🡒 Instructor tab 🡒 Student Admin 🡒 View gradebook. The URL should be: ``http(s)://{{ MFE_HOST }}/gradebook/{{ course ID }}``. When running locally, the gradebook of the demo course is available at: http://apps.local.overhang.io/gradebook/course-v1:edX+DemoX+Demo_Course
+
+Profile
```

Comment:
Note that the Profile MFE still requires the SiteConfiguration "ENABLE_PROFILE_MICROFRONTEND":true

Reply:
Right. This is a pain in the butt. I'll add a `site-configuration set/unset` script to the openedx Docker image.